Flixel Features/SetTileProperties/source/PlayState.hx

```diff
@@ -0,0 +1,134 @@
+package;
+
+import flixel.addons.editors.ogmo.FlxOgmoLoader;
+import flixel.effects.particles.FlxEmitter;
+import flixel.effects.particles.FlxParticle;
+import flixel.FlxG;
+import flixel.FlxObject;
+import flixel.FlxSprite;
+import flixel.FlxState;
+import flixel.text.FlxText;
+import flixel.tile.FlxTile;
+import flixel.tile.FlxTilemap;
+import flixel.ui.FlxButton;
+import flixel.math.FlxMath;
+import flixel.util.FlxColor;
+
+class PlayState extends FlxState
+{
+	
+	private var _map:FlxTilemap;
+	private var _leftEmitter:FlxTypedEmitter<FlxParticle>;
+	private var _rightEmitter:FlxTypedEmitter<FlxParticle>;
+	
+	private var _scores:Array<Int> = [0, 0];
```

Comment:
It seems like overkill to define two constants and an array for what would otherwise only require two ints (`scoreBlue`, `scoreRed`). Or am I missing something?